web/packages/extension/src/content.ts

```diff
@@ -209,5 +142,6 @@ function isXMLDocument(): boolean {
             unmuteOverlay: options.autostart ? "hidden" : "visible",
             splashScreen: !options.autostart,
         },
+        publicPath: utils.runtime.getURL("/dist/"),
```

Comment:
Pass the full URL of the extension's /dist/ folder to the ruffle content script.